bundles/org.openhab.binding.zwavejs/README.md

```diff
@@ -0,0 +1,91 @@
+# Z-Wave JS Binding
+
+The `zwavejs` binding integrates Z-Wave JS with openHAB, allowing you to control and monitor Z-Wave devices using the Z-Wave JS Webservice (part of Z-Wave JS UI).
+This binding supports a wide range of Z-Wave devices, including sensors, switches, dimmers, and more.
+For documentation about Z-Wave JS UI, please visit <https://zwave-js.github.io/zwave-js-ui/>.
+
+## Prerequisites
+
+Before using the `zwavejs` binding, ensure the following prerequisites are met:
+
+- You have a running instance of [Z-Wave JS](https://zwave-js.github.io/zwave-js-ui/).
+- Your Z-Wave controller is properly connected and recognized by Z-Wave JS.
+- The Z-Wave JS instance has the Webservice enabled (Settings -> Home Assistant -> WS Server).
+- Network connectivity exists between openHAB and the Z-Wave JS Webservice (the hostname and port must be reachable from openHAB).
+
+**Note**: If you are transitioning from the native Z-Wave binding to the `zwavejs` binding, you can safely test this binding without permanently affecting your setup. All node information will remain on your controller as usual. However, please avoid performing a manual reset unless absolutely necessary, as it may complicate reverting to the native binding.
+
+## Supported Things
+
+This binding supports the following types of things:
+
+- `gateway`: Represents the Z-Wave JS Webservice bridge. This is required to communicate with the Z-Wave network.
+- `node`: Represents a Z-Wave device (node) in the network. Each node can have multiple channels corresponding to its capabilities.
+
+**Note**: This binding does not maintain a Z-Wave device database and relies on the external Z-Wave JS project for device compatibility and functionality.
+
+## Discovery
+
+The `zwavejs` binding supports auto-discovery of Z-Wave devices.
+When the bridge is added and connected, it will automatically discover and add Z-Wave nodes to the openHAB system.
+The following discovery features are available:
+
+- Automatic discovery of Z-Wave nodes when they are added to the network.
+- Automatic update of node information when the node is updated in the Z-Wave network.
+
+## Bridge Configuration
+
+The `zwavejs` binding requires configuration of the bridge to connect to the Z-Wave JS Webservice.
+The configuration options include:
+
+| Name                  | Type    | Description                                         | Default | Required | Advanced |
+|-----------------------|---------|-----------------------------------------------------|---------|----------|----------|
+| hostname              | text    | Hostname or IP address of the server                | N/A     | yes      | no       |
+| port                  | number  | Port number to access the service                   | 3000    | yes      | no       |
```

Comment:
Ultra minor and not a strict request of change: I prefer the descriptions in your XML for both parameters

Reply:
I can understand the preferences. The XML leaves a bit more space to describe it, so i'll leave it as is.